core/server/middleware/api/version-match.js

```diff
@@ -0,0 +1,16 @@
+var errors = require('../../errors');
+
+function checkVersionMatch(req, res, next) {
+    var requestVersion = req.get('X-Ghost-Version'),
+        currentVersion = res.locals.safeVersion;
+
+    if (requestVersion && requestVersion !== currentVersion) {
+        return next(new errors.VersionMismatchError(
+            'Request for version ' + requestVersion + ' does not match current version ' + currentVersion
```

Comment:
suggestion: translation key?i know this error is only returned to the admin UI and https://github.com/TryGhost/Ghost/issues/3801 says its low prio. so i think it's up to you